src/components/app/app.ts

```diff
@@ -1,17 +1,25 @@
 import AppController from '../controller/controller';
 import { AppView } from '../view/appView';
+import { IDataSources } from '../view/sources/sources';
 
 class App {
+    controller: AppController;
+    view: AppView;
+
     constructor() {
         this.controller = new AppController();
         this.view = new AppView();
     }
+    
+    // writeNews (data: ) {
+    //     this.view.drawNews(data)
+    // }
```

Comment:
It's a bad practice to leave a commented-out code.
Please remove it from all files.

Reply:
Removed all commented-out code.